integration_test/third_party_apps_test/applications/apache/enable

```diff
@@ -30,5 +38,5 @@ logging:
           - apache_error
 EOF
 
-sudo service google-cloud-ops-agent restart
-sleep 60
+
```

Comment:
nit: Can we remove the extra lines and on the other files that they got left.

Reply:
Done